packages/app-store/applecalendar/pages/setup/index.tsx

```diff
@@ -70,23 +70,26 @@ export default function AppleCalendarSetup() {
                     required
                     type="text"
                     {...form.register("username")}
-                    label="Username"
+                    label={t("username")}
                     placeholder="rickroll"
                   />
                   <TextField
                     required
                     type="password"
                     {...form.register("password")}
-                    label="Password"
+                    label={t("password")}
                     placeholder="•••••••••••••"
                     autoComplete="password"
                   />
                 </fieldset>
 
                 {errorMessage && <Alert severity="error" title={errorMessage} className="my-4" />}
-                <div className="mt-5 sm:mt-4 sm:flex sm:flex-row-reverse">
+                <div className="mt-5 space-x-2 space-x-reverse sm:mt-4 sm:flex sm:flex-row-reverse">
+                  <Button type="button" onClick={() => router.back()}>
+                    {t("cancel")}
+                  </Button>
                   <Button type="submit" loading={form.formState.isSubmitting}>
-                    Save
+                    {t("save")}
                   </Button>
```

Comment:
where does this flex-row-reverse come from all the time? lets get rid of this. likely copied from an old component.

Cancel should always be left, with `color="secondary"`  never two primary buttons

Reply:
just in case cc @Jaibles